apps/redi-connect/src/pages/front/signup/SignUp.tsx

```diff
@@ -95,6 +92,12 @@ export default function SignUp() {
           lastName: values.lastName,
           userType: type.toUpperCase() as UserType,
           rediLocation: envRediLocation() as RediLocation,
+          ...(type === 'mentor'
```

Comment:
Uhhh this must be the fanciest spread I've seen in my entire career! Ternary operator inside of a spread?? 🤯

![mind blowing](https://media.giphy.com/media/26ufdipQqU2lhNA4g/giphy.gif)